src/pkg/data/consumers/governor/exporter.go

```diff
@@ -0,0 +1,66 @@
+package consumers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	api "github.com/vmware-tanzu/cloud-native-security-inspector/src/api/v1alpha1"
+	"github.com/vmware-tanzu/cloud-native-security-inspector/src/lib/log"
+	openapi "github.com/vmware-tanzu/cloud-native-security-inspector/src/pkg/data/consumers/governor/go-client"
+	"net/http"
+)
+
+type GovernorExporter struct {
+	Report    *api.AssessmentReport
+	ClusterID string
+	ApiToken  string
+	ApiClient *openapi.APIClient
+}
+
+// SendReportToGovernor is used to send report to governor url http end point.
+func (g GovernorExporter) SendReportToGovernor() error {
+	// Get governor api request model from assessment report.
+	kubernetesCluster := getGovernorAPIPayload(*g.Report)
+
+	apiSaveClusterRequest := g.ApiClient.ClustersApi.UpdateTelemetry(context.Background(), g.ClusterID).KubernetesTelemetryRequest(kubernetesCluster)
+
+	// Call api cluster to send telemetry data and get response.
+	response, err := g.ApiClient.ClustersApi.UpdateTelemetryExecute(apiSaveClusterRequest)
+	if err != nil {
+		log.Errorf("Governor api response error: %v", err)
+		return err
+	}
+	if response.StatusCode != http.StatusOK {
+		log.Errorf("Governor api response status: %v", response.StatusCode)
+		return errors.New(fmt.Sprintf("Governor api response status: %s", response.Status))
+	}
+
+	return nil
+}
+
+// getGovernorAPIPayload is used to map assessment report to client model.
+func getGovernorAPIPayload(doc api.AssessmentReport) openapi.KubernetesTelemetryRequest {
+	kubernetesCluster := openapi.NewKubernetesTelemetryRequestWithDefaults()
+	for _, nsa := range doc.Spec.NamespaceAssessments {
+		for _, workloadAssessment := range nsa.WorkloadAssessments {
+			kubernetesWorkloads := openapi.NewKubernetesWorkloadWithDefaults()
+			kubernetesWorkloads.Name = workloadAssessment.Workload.Name
+			kubernetesWorkloads.Kind = workloadAssessment.Workload.Kind
+			if nsa.Namespace.Name != "" {
```

Comment:
This check is redundant

If nsa.Namespace.Name  is empty, then kubernetesWorkloads.Namespace = nsa.Namespace.Name results to empty.

And defualt value of kubernetesWorkloads.Namespace is also empty.


You can just replace it with `kubernetesWorkloads.Namespace = nsa.Namespace.Name`